### PR TITLE
Find.test.ts

### PR DESCRIPTION
**Pull request by gcrev93**
Created a test for the find helper functions.

Request:

I would like edit the find.ts functions to check if the slotID exists. Example:
```
export const findIndexOfSlotIdsBySlotId =
  (array: SlotId[], slotId: SlotId): number => array.findIndex((arrId) => {
    const index = arrId.abilityName === slotId.abilityName && arrId.slotName === slotId.slotName 
    return  { index > -1 ? index :  throw new Error('Cannot find slotId ${slotId} in array') }
  })
```

In the current test i check for specific values. I would rather keep the same format as other test and check for an error

### CODE REVIEW

**Comment by howlowck** [comment]
Hey @gcrev93 any function that starts with `find` means that it fails gracefully if the function does not find anything.  So it should not throw an error.

File: src/tests/unit/helpers/find.test.ts [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+interface MockUserState {
+    firstName: string,
+    lastName: SVGAnimatedString,
+    age: number
+}
+
+describe('findIndexofSlotIdsBySlotId function', () => {
+    test('when SlotID does not exist', () => {
+        const abilities: Ability<MockUserState, AllSyncStorageLayer<MockUserState>>[] = [{
```
Comment (howlowck) [inline src/tests/unit/helpers/find.test.ts]: Remove `abilities`

File: src/tests/unit/helpers/find.test.ts [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+            abilityName: 'ability4'
+        }
+
+        const actual = findHelpers.findIndexOfSlotIdsBySlotId(slotIds, singleSlotId)
+        expect(actual).toEqual(-1)
+        //expect(() => findHelpers.findIndexOfSlotIdsBySlotId(slotIds, singleSlotId)).toThrow()
+    })
+
+    test('when SlotID does exist', () => {
+        const abilities: Ability<MockUserState, AllSyncStorageLayer<MockUserState>>[] = [{
```
Comment (howlowck) [inline src/tests/unit/helpers/find.test.ts]: Remove `abilities`

**Review by howlowck — CHANGES_REQUESTED** [review]
Remove `abilities`

File: src/tests/unit/helpers/find.test.ts (hunk 1 above)
Comment (gcrev93) [reply src/tests/unit/helpers/find.test.ts]: removed abilities and unnecessary comments

**Review by howlowck — APPROVED** [review]

**Comment by howlowck** [comment]
approved!

**Pull request merged**